_data/simple-icons.json

```
@@ -13645,6 +13645,11 @@
             "hex": "FF0000",
             "source": "https://partnermarketinghub.withgoogle.com/#/brands/"
         },
+        {
+            "title": "Yr",
+            "hex": "00B8F1",
```

```suggestion
            "hex": "00B9F1",
```
Taken from their logo on their website